benchmark/crypto/randomBytes.js

```diff
@@ -5,7 +5,7 @@ const { randomBytes } = require('crypto');
 
 const bench = common.createBenchmark(main, {
   size: [64, 1024, 8192, 512 * 1024],
-  n: [1e3],
+  n: [1e4],
```

Comment:
Can you update this cases to:

```suggestion
  size: [64, 1024, 8 * 1024, 16 * 1024],
  n: [1e5],
```

I don't think that worth having such a high number in size, and I think is more worth having more iterations.

With these numbers, we still keep each run below 3s, with `512 * 1024` and `1e5`, it takes `19s`.

/cc @nodejs/performance In case someone has a different opinion about this.